qa/rpc-tests/test_framework/util.py

```diff
@@ -338,7 +339,10 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
         binary = os.getenv("BITCOIND", "bitcoind")
     args = [ binary, "-datadir="+datadir, "-server", "-keypool=1", "-discover=0", "-rest", "-mocktime="+str(get_mocktime()) ]
     if extra_args is not None: args.extend(extra_args)
-    bitcoind_processes[i] = subprocess.Popen(args)
+    if stderr is not None:
```

Comment:
I don't think this if/else construction is needed. subprocess.Popen's stderr argument defaults to None, so you can just call `bitcoind_processes[i] = subprocess.Popen(args, stderr=stderr)`